src/sm64ap.cpp

```diff
@@ -16,9 +16,12 @@ extern "C" {
 #include <set>
 
 #define WARP_NODE_CREDITS_MIN 0xF8 // level_update.c
+#define NUM_PAINTING_LOCKS 15
 
 // Set to false on some branch for compat with patches
 static constexpr bool SM64AP_SUPPORT_MOVE_RANDO = true;
+// Set to false on some branch for compat with patches
+static constexpr bool SM64AP_SUPPORT_PAINTING_RANDO = true;
```

Comment:
I'm guessing you added this because of the move rando option. Thats just something I forgot to remove though, so you don't need to add it here. I'll remove the move rando one later